### ISSUE TITLE
support SSR

### ISSUE THREAD

**Issue by ghost** [storeon/router#33]
Right now this module can't be used on server for SSR because it relies on window object and location object
and doesn't provide any alternative for server

**Comment by ivan-solovev** [comment]
I agree. If it is a separate file for the server with the same public API. Is this normal?

**Comment by ghost** [comment]
> I agree. If it is a separate file for the server with the same public API. Is this normal?

Creating the same routes two times: one on server and one on client should not be considered normal

I'm ok to have different code on server than client, just not redefine routes
